install/htaccess.dist

```diff
@@ -1,5 +0,0 @@
-<IfModule mod_rewrite.c>
```

Comment:
Shouldn't we check first, that it still follows the requirements for DynDNS? Or are there no standards?